package.json

```diff
@@ -1,6 +1,11 @@
 {
   "name": "blueprint-ui",
-  "version": "1.1.2",
+  "version": "1.2.0-rc1",
+  "license": "MIT",
+  "config": {
+    "openapiArgs": "--additional-properties ngVersion=15.1 --additional-properties modelPropertyNaming=original --type-mappings=DateTime=Date",
+    "deleteGeneratedCommand": "rimraf src/app/generated/vm-api/"
```

Comment:
The delete line shoudl be:

"deleteGeneratedCommand": "rimraf src/app/generated/blueprint.api/"

Reply:
another good catch!